tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/DiagnosticVerifier.cs

```diff
@@ -32,10 +32,7 @@ internal void VerifyCSharpDiagnostic(string source, params DiagnosticResult[] ex
         /// </summary>
         /// <returns>The diagnostic analyzer being tested.</returns>
         [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "It's not appropriate here")]
-        protected virtual DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
-        {
-            return null;
-        }
+        protected abstract DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer();

```

Comment:
Avoids cryptic error if you forget to override. Wasn't possible before because both `GetCSharpDiagnosticAnalyzer` and `GetVisualBasicDiagnosticAnalyzer` existed, but now only one of them exists based on compilation symbols.